src/proto/Binance.proto

```diff
@@ -75,6 +75,40 @@ message TokenUnfreezeOrder {
     int64 amount = 3;  // amount of token to unfreeze
 }
 
+message HTLTOrder {
+    // 0xB33F9A24      // prefix
+    bytes from = 1;    // signer address
+    bytes to = 2;    // recipient address
+    string recipientOtherChain = 3;
+    string senderOtherChain = 4;
+    bytes randomNumberHash = 5;
```

Comment:
can you use snake case as suggested by guidelines: https://developers.google.com/protocol-buffers/docs/style ?